skimage/measure/_regionprops.py

```diff
@@ -938,6 +938,11 @@ def regionprops_table(
     The table is "tidy" [1]_; it is a dictionary mapping property names to
     value arrays. See Notes section below for details.
 
+    This function is typically useful when you have a pre-determined list of
+    region properties you want to compute (and possibly print or feed). If you
+    can afford to compute given region properties only when they are needed,
+    consider using :func:`skimage.measure.regionprops` instead.
```

Comment:
I don't think this really captures why you would use a table or regionprops. I'm struggling to come up with something better but:

> Use this function when you want to do downstream (statistical) data analysis, or save region data to disk in a structured way. One downside of this function is that it breaks multi-dimensional properties into independent columns; for example, the region centroids of a 3D image are placed into three columns, one per dimension. When you need to do complex computations on the region properties, using :func:`skimage.measure.regionprops` might make more sense.

Reply:
I like it... We might as well write out what we mean, as in "save region data to disk in a structured way" (I tried being elliptic and concise with "print or feed" but then it's not clear!). That was exactly what the user wanted to do (write region data to a CSV file) when they reported the coords issue in the image.sc forum.

I'll digest your blurb and come up with something...

Reply:
Ok, see a639075389b058c.